docs/run-book.md

```diff
@@ -0,0 +1,177 @@
+Run Book
```

Comment:
Is it worth noting (generally, or for specific sections) that this is derived from the SSPP and including a link to wherever that will live? It's unlikely that somebody would try to do this unilaterally, but it would clarify that updating some of these policies is a more complex process than editing this file.

Reply:
I like that idea a lot. Will update in a new PR alongside some enhancements related to new audit logging stuff

Reply:
#247